jenkins/build.sh

```diff
@@ -15,22 +15,27 @@
 # about this software or licensing, please email opensource@seagate.com or
```

Comment:
`conf/script/v2/ha_setup.py`

you need to add all py file in cortx-ha/ha only else they will not able to communicate package

cc: @indrajitzagade

Reply:
This python file is not required in packaging, this will used by the cortx-provisioner to do mini provisioner actions.